.github/workflows/build-pr-preview.yml

```diff
@@ -5,11 +5,13 @@ on:
     # To manage 'surge-preview' action teardown, add default event types + closed event type
     types: [opened, synchronize, reopened, closed]
     paths:
-      - 'modules/ROOT/**'
+      - 'modules/**'
       - 'antora.yml'
       - '.github/workflows/build-pr-preview.yml'
 jobs:
   build_preview:
+    # Don't build preview for non member of the repository
+    if: ${{ secrets.SURGE_TOKEN_DOC != '' }}
```

Comment:
This doesn't work because here, the workflow cannot access to the env or secrets context
We will run another job that will expose a boolean as job output, and the output will be available here